processor/src/main/java/com/linecorp/decaton/processor/runtime/internal/PartitionContexts.java

```diff
@@ -53,6 +57,8 @@ public class PartitionContexts implements OffsetsStore, AssignmentStore, Partiti
     private final Map<TopicPartition, PartitionContext> contexts;
 
     private final AtomicBoolean reloadRequested;
```

Comment:
`reloadRequested` is useful for quickly determining if contexts should be reloaded without iteration.

Reply:
Hm, IMO having duplicated info (with per-partition reload) could be an error prone when we maintain this code in the future, while iterating over partitions unlikely becomes a bottleneck (since we already doing iterating partitions for updating high watermarks, checking partition needs pause/resume) so the benefit to have `reloadRequested` separately is small.